include/dpp/channel.h

```diff
@@ -494,6 +512,13 @@ class DPP_EXPORT channel : public managed, public json_interface<channel>  {
 	 */
 	channel_type get_type() const;
 
+	/**
+	 * @brief Get the default forum layout type used to display posts in forum channels
+	 *
+	 * @return forum_layout_types Forum layout type
+	 */
+	forum_layout_type get_default_forum_layout() const;
```

Comment:
should we name this `get_default_forum_layout_type`? Would be longer but more descriptive